src/dswx_sar/defaults/algorithm_parameter_s1.yaml

```diff
@@ -126,11 +126,16 @@ runconfig:
             line_per_block: 400
 
         masking_ancillary:
+            land_cover_darkland_list: ['Bare sparse vegetation', 'Urban', 'Moss and lichen']
+            land_cover_darkland_extension_list: ['Grassland', 'Shrubs']
```

Comment:
It would be great if there is some explanation about `land_cover_darkland_extension_list` as a comment around this line.